lms/services/course.py

```diff
@@ -54,9 +90,12 @@ def _create(self, authority_provided_id):
             course_settings.set("canvas", "sections_enabled", False)
 
         course = Course(
-            consumer_key=self._consumer_key,
+            application_instance_id=self._application_instance.id,
```

Comment:
Use the instance directly here, not the ID (to allow this to work for newly created AIs